### DISCUSSION TITLE
Is there a way to manually set the pos tag of a certain token in advance?

### DISCUSSION THREAD

**Discussion by SilasTHU** [explosion/spaCy#10525] [Help: Coding & Implementations] [answered]
I want to specify some words with manual labeled pos tags. And SpaCy do the rest for me based on that. Is that possible?

Thank you for your help.

**Comment by polm** [comment, marked as answer]
Yes, you can do that in a straightforward way. 

```
import spacy
nlp = spacy.load("en_core_web_sm")
doc = nlp.make_doc("I like cheese") # just tokenize
doc[0].tag_ = "VBP" # "I" is now a verb
doc2 = nlp(doc) # this predicts everything but doesn't overwrite existing settings
```

Note this depends on the `overwrite` flag in the `Tagger` and also possibly the AttributeRuler. See [the docs](https://spacy.io/api/tagger#config).

**Reply by adrianeboyd** [reply]
But note that the tagger component won't take this manual tag into account in its predictions. It will make the same predictions for the surrounding words as if the tag weren't there, so you can't use this approach to change the tagger's output.

**Reply by SilasTHU** [reply]
Thank you! It's great!!!
